tests/dummy/config/targets.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const browsers = [
-	'last 1 Chrome versions',
+	'chrome >= 30',
```

Comment:
Why do you need this change in development?

Reply:
Oops. I don’t need it